Test/TestConfigurationUtils.ps1

```diff
@@ -35,6 +36,14 @@ function Test-IsProcessRunning {
     return [bool] $Proc
 }
 
+function Assert-AreDLLsPresent {
+    $MissingDLLsErrorReturnCode = [int64]0xC0000135
```

Comment:
Please paste comment with link to https://msdn.microsoft.com/en-us/library/cc704588.aspx and describe that this error is `STATUS_DLL_NOT_FOUND`